index.js

```diff
@@ -88,6 +88,10 @@ function Browserify (files, opts) {
     self._bresolve = browserField === false
         ? function (id, opts, cb) {
             if (!opts.basedir) opts.basedir = path.dirname(opts.filename)
+            // Resolve builtin modules.
+            if (self._mdeps.options.modules[id]) return process.nextTick(function () {
```

Comment:
this should use resolve’s isCore functionality, i think?

Reply:
@ljharb I don't believe so, as custom built-ins can be fed into Browserify, and those won't be picked up by `resolve.isCore`.  See: https://github.com/browserify/browserify/blob/master/index.js#L551-L553

Reply:
If those can *override* the default ones, then that's a fair point - but then browserify should be using resolve's node-version-specific logic for the defaults, which it doesn't appear to be.

Reply:
@ljharb Yes, it overrides the defaults, in a way.  It's just a direct replacement of built-in resolution.  Browserify doesn't use the node-version-specific logic, and that's a separate issue from this one that's worth logging.  I'm maintaining a fork right now with this change, and I'd like to be able to depend on the officially published version.  Is there anything else holding this up or can it move forward?